src/components/shared/BankData.vue

```diff
@@ -1,5 +1,6 @@
 <template>
 	<ul class="bank-data-content">
+		<li><h2> {{ $t( 'donation_confirmation_payment_bank_transfer_details' ) }} </h2></li>
```

Comment:
The `BankData` component is also used in the sidebar of the donation page, so you should move the title out of it and into `SuccessMessageBankTransfer`.